pkg/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -307,7 +307,14 @@ func (b *builder) imagePullSecrets() []corev1.LocalObjectReference {
 		return []corev1.LocalObjectReference{}
 	}
 
-	return []corev1.LocalObjectReference{{Name: b.dk.PullSecretName()}}
+	imagePullSecrets := make([]corev1.LocalObjectReference, 0)
```

Comment:
Same code snippet is used in StatefulSet creation. Maybe worth putting somewhere in a comon place.

Reply:
```
func (dk *DynaKube) ImagePullSecrets() []corev1.LocalObjectReference {
  ...
}
```